staging/src/k8s.io/csi-api/pkg/crd/crd.go

```diff
@@ -14,106 +14,9 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
+// Package crd is only for running tests to verify the manifest files
```

Comment:
I want the manifests dir to contain only the manifests and nothing else so that people can grab the entire directory as a source of truth and not get test files and other stuff with it. Would prefer to leave this as is.